stestr/selection.py

```diff
@@ -83,24 +83,28 @@ def _get_regex_from_whitelist_file(file_path):
             try:
                 lines.append(re.compile(line_regex))
             except re.error:
-                print("Invalid regex: %s in provided whitelist file" %
+                print("Invalid regex: %s in provided inclusion_list file" %
                       line_regex, file=sys.stderr)
                 sys.exit(5)
     return lines
 
 
 def construct_list(test_ids, blacklist_file=None, whitelist_file=None,
-                   regexes=None, black_regex=None):
+                   regexes=None, black_regex=None, exclusion_list_file=None,
+                   inclusion_list_file=None, exclusion_regex=None):
     """Filters the discovered test cases
 
     :param list test_ids: The set of test_ids to be filtered
-    :param str blacklist_file: The path to a blacklist file
-    :param str whitelist_file: The path to a whitelist file
+    :param str blacklist_file: DEPRECATED: Replaced by exclusion_list_file
+    :param str whitelist_file: DEPRECATED: Replaced by inclusion_list_file
```

Comment:
I think you can remove the deprecated args here, we don't treat `construct_list` as a Stable API. See https://stestr.readthedocs.io/en/latest/api.html#internal-apis

Reply:
Changing "deprecation note" to simple mention of new args...